taxi/migrations/0001_initial.py

```diff
@@ -1,66 +1,181 @@
+                (
+                    "date_joined",
+                    models.DateTimeField(
+                        default=django.utils.timezone.now, verbose_name="date joined"
+                    ),
+                ),
+                ("license_number", models.CharField(max_length=255, unique=True)),
+                (
+                    "groups",
+                    models.ManyToManyField(
+                        blank=True,
+                        help_text="The groups this user belongs to. A user will get all permissions granted to each of their groups.",
+                        related_name="user_set",
+                        related_query_name="user",
+                        to="auth.group",
+                        verbose_name="groups",
+                    ),
+                ),
+                (
+                    "user_permissions",
+                    models.ManyToManyField(
+                        blank=True,
+                        help_text="Specific permissions for this user.",
+                        related_name="user_set",
+                        related_query_name="user",
+                        to="auth.permission",
+                        verbose_name="user permissions",
+                    ),
+                ),
             ],
             options={
-                'ordering': ['name'],
+                "verbose_name": "driver",
+                "verbose_name_plural": "drivers",
             },
+            managers=[
+                ("objects", django.contrib.auth.models.UserManager()),
+            ],
         ),
         migrations.CreateModel(
-            name='Car',
+            name="Car",
             fields=[
-                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('model', models.CharField(max_length=255)),
-                ('drivers', models.ManyToManyField(related_name='cars', to=settings.AUTH_USER_MODEL)),
-                ('manufacturer', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='taxi.manufacturer')),
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("model", models.CharField(max_length=255)),
+                (
+                    "drivers",
+                    models.ManyToManyField(
+                        related_name="cars", to=settings.AUTH_USER_MODEL
+                    ),
+                ),
+                (
+                    "manufacturer",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to="taxi.manufacturer",
+                    ),
+                ),
             ],
         ),
     ]
```

Comment:
Do not overwrite the initial migration, create a new migration apart from the existing one.

Reply:
But initial migration doesn`t working all the time.When i`m trying to run another migration nothing happens.So,i need to delete this migration and run new one, then all works